packages/blocks/package.json

```diff
@@ -50,6 +50,7 @@
 		"hpq": "^1.3.0",
 		"is-plain-object": "^5.0.0",
 		"memize": "^2.1.0",
+		"react-is": "18.2.0",
```

Comment:
We shouldn't pin the version here.

Reply:
React version are pinned, so I used same logic here.

Reply:
This is what I see:
<img width="340" alt="Screenshot 2023-12-20 at 13 14 32" src="https://github.com/WordPress/gutenberg/assets/699132/8a347fc7-4958-42f4-985b-7b3f1bcb0080">

What do I miss here in regards to pinned React version?

Reply:
![CleanShot 2023-12-21 at 10 23 46](https://github.com/WordPress/gutenberg/assets/240569/fa6d8ca0-c25a-4294-984c-498ee41d2a4a)

It is pinned in the main package file.

Reply:
Yes, we pin versions for the project. However, we should do our best to never pin them in WordPress packages.